mods/persistence/modules/chargen/defines/roles.dm

```diff
@@ -170,3 +177,12 @@
 	)
 
 	text_book_type = /obj/item/book/skill/engineering/electrical/expert
+
+/decl/hierarchy/chargen/role/educator
+	ID = "educator"
+	name = "Educator"
+	skills = list(
+		SKILL_LITERACY = 1
```

Comment:
Literacy 2? Seems rather weak in comparison to others but I'm not sure exactly what other skills would be appropriate for it.